tofu/geom/_core.py

```diff
@@ -2308,43 +2308,54 @@ def _get_largs_dsino():
     ###########
 
     def _checkformat_inputs_Struct(self, struct, err=True):
```

Comment:
It would be great to have a one-line docstring for this method. Also, just thinking that  err=True could be raise_error=True.

Reply:
Thanks, will gradually rename in future commits :-)